Network/dedicated-load-balancer-using-cli-basic-management.md

```diff
@@ -0,0 +1,65 @@
+{{{
+  "title": "Dedicated Load Balancer Basic Management Using CLI",
+  "date": "9-1-2015",
+  "author": "Mark Turpin",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Dedicated Load Balancer Basic Management Using CLI
+
+This KB will go over the basics of creating a Load Balancer VIP and Services when using the Command Line Interface (CLI) for dedicated Load Balancers
+
+### Prerequisites
+
+* HA Dedicated Load Balancers have been deployed to your account
+* Must have an Admin login to the Netscaler
+* Must be able to SSH to the RNAT IP of the HA pair (typically ends with .101)
+
+### Notes
+
+* This example will configure a 2 node web server farm that is load balancing traffic on http (port 80) and using "cookie insert" for persistence type
+* Least connection is the default load balancing algorithm
+* Ping/TCP checks are the default health monitors
+
+### Steps
+
+1. Log into your dedicated load balancer RNAT IP (typically ends with .101) using SSH and an Admin login
+2. Now in the CLI, we will create the server objects.  These will be the web servers being load balanced.
+
+  ```
+  add server VA1ACMEWEB01 10.10.10.15
+  add server VA1ACMEWEB02 10.10.10.16
+  ```
+
+3. Next, we will create a service for both servers (using HTTP)
+
+  ```
+  add service VA1ACMEWEB01_http_80 VA1ACMEWEB01 http 80
+  add service VA1ACMEWEB02_http_80 VA1ACMEWEB02 http 80
+  ```
+
+4. Next, we will create the Virtual Server.  We will use the website name and port in the name for ease of management.
```

Comment:
from a tech angle i'd note that you used the private VIP on step 4 from the pool of pre-assigned private IP's given out during the NLB provisioning.  its just not clear from the text where the IP came from.